Vishnu_45/html/Assignment4/Assign_2.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
need proper code indentation
remove all extra lines
add new line
change all with at the end of css property ends